roles/lib_openshift/library/oadm_manage_node.py

```diff
@@ -748,6 +748,29 @@ class OpenShiftCLIError(Exception):
     pass
 
 
+def locate_oc_binary():
```

Comment:
Personally I tend to dislike static methods unless they are absolutely necessary. I'm not against moving it to another location. Could it go in the same module as utils as a function?